modules/dna_rna_tools.py

```diff
@@ -0,0 +1,196 @@
+COMPLEMENT_DNA = {'a': 't', 'A': 'T', 't': 'a', 'T': 'A',
+                  'g': 'c', 'G': 'C', 'c': 'g', 'C': 'G'}
+
+COMPLEMENT_RNA = {'a': 'u', 'A': 'U', 'u': 'a', 'U': 'A',
+                  'g': 'c', 'G': 'C', 'c': 'g', 'C': 'G'}
+
+
+def check_dna(sequence: str) -> bool:
+    """
+    Checkes is input string a DNA sequence.
+    If input sequence is DNA it will return True.
+    
+    Input:
+    -sequence (str): input nucleic acid sequence, any letter case is suitable.
+    
+    Output:
+    returns True or False (bool), is nucleic acid is DNA.
+    """
+    flag = True
+    valid_nucleotides_dna = {'A', 'T', 'C', 'G'}
+    unique_chars = set(sequence.upper())
+    if not unique_chars.issubset(valid_nucleotides_dna):
+        flag = False
+    return flag
+
+
+def check_rna(sequence: str) -> bool:
+    """
+    Checkes is input string a RNA sequence.
+    
+    Input:
+    -sequence (str): input nucleic acid sequence, any letter case is suitable.
+    
+    Output:
+    returns True or False (bool), is nucleic acid is RNA.
+    """
+    flag = True
+    valid_nucleotides_rna = {'A', 'U', 'C', 'G'}
+    unique_chars = set(sequence.upper())
+    if not unique_chars.issubset(valid_nucleotides_rna):
+        flag = False
+    return flag
+
+
+def reverse(*sequences: list) -> list:
+    """
+    Creates reverse sequence(s) of RNA or DNA. Function accepts one or arbitary arguments - nucleic acid sequences,
+    and returns string if one argument was entered or list for two and more arguments.
+    
+    Input:
+    - *sequences list[str]: list of DNA or RNA sequences.
+    
+    Output:
+    list[str] or str of reversed sequence(s).
+    """
+    reverse_sequences = []
+    for sequence in sequences:
+        if check_dna(sequence) or check_rna(sequence) is True:
```

Comment:
```suggestion
        if check_dna(sequence) or check_rna(sequence):
```
Тут не нужна проверка на то какой bool твой bool чтобы использовать его в bool-условии. Просто сразу if ...: